### DISCUSSION TITLE
Pairing zigbee router with zigbee2mqtt

### DISCUSSION THREAD

**Discussion by ckegels** [syssi/esphome-zb-gw03#38] [Q&A]
i flashed all zigbee router firmware and files but cant seem to get it to connect to zigbee2mqtt
(it is found but it disconnects right after connecting (also states not supported))

**Comment by ckegels** [comment]
after a couple hours of trying i was able to pair it (tho it really did not want to) it would not pair with my coordinator or when i used all devices it also did not work i found one light bulb on my network that for some reason it would use to connect.

**Reply by syssi** [reply]
I don't have much experience with the router firmware and zigbee2mqtt. Did you use the mentioned firmware of the router documentation or did you choose another version/file?
